ffi/koptcontext.lua

```diff
@@ -1,3 +1,57 @@
+--[[--
+# Leptonica cheatsheet:
+## Data structures:
+`PIX - basic data structure - stores image`
+
+`PIXA - array of PIX`<br/>
+`BOX - stores rectangle (x, y, w, h)`<br/>
+`BOXA - array of BOX`<br/>
+`SEL - selector for morphological operations`<br/>
+`SELA - array of SEL`<br/>
+`SELA - array of SEL`<br/>
+`NUMA - array of numbers`<br/>
```

Comment:
I'd say if you want it to look like code indent it by four spaces, and if you want it to be an HTML list… well, use a Markdown list. :-)

```
* `BOX` - stores things
* `BOXA` - array of BOX
```

Or something like that.